slack-lint/src/main/java/slack/lint/resources/ImportAliasesLoader.kt

```diff
@@ -0,0 +1,82 @@
+package slack.lint.resources
+
+import com.android.tools.lint.detector.api.Context
+import java.io.File
+import java.io.StringReader
+import java.util.Properties
+
+object ImportAliasesLoader {
+    const val PROPERTY_FILE = "gradle.properties"
+    const val IMPORT_ALIASES_PROPERTY = "nonTransitiveRClass.importAliases"
+
+    private const val TOP_LEVEL_FILENAME = "settings.gradle"
+
+    // Caches the import aliases to improve performance.
+    var importAliases = emptyMap<String, String>()
```

Comment:
Static caching like this is an antipattern in lint, let's remove it unless you can offer a measurable performance benefit gained by this to justify it.

Reply:
This was to alleviate the slowdown resulting from `findRoot`. If we're removing that then I'll remove this as well.